src/commands/synthetics/__tests__/fixtures.ts

```diff
@@ -569,16 +573,18 @@ export const getTestPayload = (override?: Partial<TestPayload>) => ({
   ...override,
 })
 
-export const getMobileVersion = (override?: Partial<MobileApplicationVersion>) => ({
-  id: '123-abc-456',
-  application_id: '789-dfg-987',
-  file_name: 'bla.',
-  original_file_name: 'test.apk',
-  is_latest: true,
-  version_name: 'test version',
-  created_at: '22-09-2022',
-  ...override,
-})
+export const getTestAndConfigOverride = (appId: string): TestWithOverride => {
+  const testWithOverride = {test: getMobileTest(), overriddenConfig: getTestPayload()}
+  testWithOverride.test.options.mobileApplication!.applicationId = appId
+
+  return testWithOverride
+}
```

Comment:
**nit** I find it weird to call this function `getTestAndConfigOverride()` when it's actually made for mobile unit tests. Can you rename it, and avoid using a non-null assertion by adding an optional `appId = 'mobileAppUuid'` argument to `getMobileTest()`?
```suggestion
export const getMobileTestWithOverride = (appId: string): MobileTestWithOverride => {
  return {
    test: getMobileTest('abc-def-ghi', appId),
    overriddenConfig: getTestPayload(),
  }
}
```